### PR TITLE
[Merged by Bors] - Cleanup of repo root and README

### PR DESCRIPTION
**Pull request by KaiserKarel**
Clearly specifies who the owners of some of the `CAPS` files are. Working on changing the README.md too, and want to make sure that technical-writing has a look at that too.

rendered link: https://github.com/ComposableFi/composable/tree/karel/update-readme

### CODE REVIEW

**Review by dzmitry-lahoda — DISMISSED** [review]
changes do improve, so delete some. assuming that some is outlined in dockerfiles and CI files and may be in other readmes.

**Review by hussein-aitlahcen — DISMISSED** [review]

**Review by Martin-Kiss — DISMISSED** [review]
LGTM

**Review by vimukthi-git — APPROVED** [review]

**Review by Martin-Kiss — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by hussein-aitlahcen — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]
likely banner was published somewhere and can be just link?

File: README.md [hunk 1]
```diff
@@ -1,57 +1,32 @@
...
+[bors-badge]: https://bors.tech/images/badge_small.svg
+[bors-url]: https://app.bors.tech/repositories/45659
 
+## Documentation
 
-## Run  
-
-You may run [Composable parachain with relay chain](scripts/polkadot-launch).
-
-Or via `nix develop .#devnet`.
```
Comment (dzmitry-lahoda) [inline README.md:47]: assumption we will bring nix soon to run stuff back into dev docs

**Review by benluelo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (KaiserKarel) [reply README.md:47]: Yeah, just removing that from the README now until we have changed to nix altogether.

**Comment by KaiserKarel** [comment]
> likely banner was published somewhere and can be just link?

AFAIK for this one no; and tbh would prefer for the repo to be internally consistent.

**Pull request closed without merging**